data/styles/vehicle/include/Basemap.mapcss

```diff
@@ -233,9 +233,9 @@ line|z11-12[waterway=riverbank],
 line|z13[waterway=river],
 line|z13[waterway=riverbank],
 {width: 1.6;}
-/*line|z13[waterway=stream],
-line|z13[waterway=canal],
-line|z13[natural=strait],
+/*line|z16[waterway=stream],
+line|z16[waterway=canal],
+line|z16[natural=strait],
```

Comment:
No reason to modify here, because it is commented and below there are also commented styles z14- ..

Reply:
thanks, fixed!